src/components/search-form/SearchForm.js

```diff
@@ -152,13 +163,24 @@ class SearchForm extends React.Component<Props, State> {
 
         const SearchActions = () => (
             <div className="action-buttons">
-                <button
-                    type="button"
-                    className="action-button search-button"
-                    title={formatMessage(messages.searchButtonTitle)}
-                >
-                    <Search16 />
-                </button>
+                {onSubmit ? (
+                    <button
+                        type="button"
+                        className="action-button search-button"
+                        title={formatMessage(messages.searchButtonTitle)}
+                    >
+                        <Search16 />
+                    </button>
+                ) : (
+                    <div
+                        type="button"
```

Comment:
Can we omit `type` and `title`, since this variation is a purely visual treatment?